.yarn/plugins/@yarnpkg/plugin-compat.cjs

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: `@yarnpkg/plugin-compat`,
+  factory: (require) => {
+    // we are not using PNP and want to use `typescript@next` in CI without hassle
+    // dummy implementation to override the built-in version of this plugin
```

Comment:
In `>=3.0.0-rc.1` the patch is optional and wont stop the install if it fails

```
yarn set version 3.0.0-rc.5
```

Reply:
I'll stay on the stable version right now, but noted - I'll remove it once we switch over :+1: